source/appModules/winword.py

```diff
@@ -14,6 +14,32 @@
 from NVDAObjects.IAccessible.winword import WordDocument as IAccessibleWordDocument
 from NVDAObjects.UIA.wordDocument import WordDocument as UIAWordDocument
 from NVDAObjects.window.winword import WordDocument
+from utils.displayString import DisplayStringIntEnum
+
+
+class ViewType(DisplayStringIntEnum):
+	"""Enumeration containing the possible view types in Word documents."""
```

Comment:
Could you add a link to documentation of these values?

Reply:
Thanks @SaschaCowley . I've done it now.